test/types/tuple/split-init/multipleInit.future

```diff
@@ -0,0 +1,5 @@
+bug: tuple destructuring split initialization gives questionable error message
+
```

Comment:
I think the second line should just be
```
#16086
```

and then the rest of the file empty. You can put commentary in the issue itself.

I'd file this as a "feature request" rather than a "bug"